README.md

```diff
@@ -275,6 +283,58 @@ connector.connect(
 )
 ``` 
 
+### Async Driver Usage
+The Cloud SQL Connector for Python currently supports the
+[asyncpg](https://magicstack.github.io/asyncpg) Postgres database driver.
+This driver leverages [asyncio](https://docs.python.org/3/library/asyncio.html)
+to improve the speed and efficiency of database connections through concurrency.
+
+The Cloud SQL Connector has an async `create_async_connector` that can be used
```

Comment:
It's maybe worth calling out the exact behavior -- that using `create_async_connector` will create a Connector in the current loop, while Connector will create a new loop in a background thread for internal use. 

The 2nd might be preferable with the sync drivers in an async context.